temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -435,9 +444,10 @@ def with_child_unrestricted(self, *child_path: str) -> SandboxMatcher:
     # Due to a metaclass conflict in sandbox, we need zipfile module to pass
     # through always
     "zipfile",
-    # This is a very general module needed by many things including pytest's
+    # Very general modules needed by many things including pytest's
     # assertion rewriter
     "typing",
+    "typing_extensions",
```

Comment:
Without this `TypedDict` fields are broken.

Reply:
TypedDict fields are broken for Pydantic only or in general in the SDK? For the former, I can see that if they are using `typing_extensions`, but for the latter that'd be a bit strange unless you're using `typing_extensions` as a user. Also, can you make a comment above this line just saying why it is here?

Reply:
Just for Pydantic. It caused a problem with pytest's assertion rewriter, as you noted for `typing` on the line above. There's no drawback to passing it through, right?
Added a comment.

Reply:
Right, no real drawback here (it's effectively part of stdlib IMO). We technically separated the restriction sets as "minimum", "with temporal", and "maximum" just in case people wanted to be more restrictive, but the default is "maximum" which is basically what everyone uses. You could move this from minimum to one of the others, but it doesn't really matter.